### PR TITLE
Add a method for the organization_purge action

### PR DESCRIPTION
**Pull request by nicholsn**
## Description
This PR adds the `organization_purge` action from the CKAN API. This allows a deleted organization to be fully removed from a ckan instance, which can come in handy for use cases like test scripts that create and teardown resources.

## Example
```
#' ckanr_setup(url = "https://demo.ckan.org", key=getOption("ckan_demo_key"))
#'
#' # create an organization
#' (res <- organization_create("foobar", title = "Foo bars",
#'   description = "love foo bars"))
#'
#' # delete the organization just created
#' res$id
#' organization_delete(id = res$id)
#'
#' # purge the organization just deleted
#' res$id
#' organization_purge(id = res$id)
#' }
```

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -48,7 +48,7 @@ Suggests:
     testthat,
     xml2,
     lazyeval
-RoxygenNote: 7.1.1
+RoxygenNote: 7.1.1.9000
```
Comment (sckott) [inline DESCRIPTION:51]: can you undo this change? don't want to depend on dev versions of roxygen2

File: R/organization_purge.R [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+#' ckanr_setup(url = "https://demo.ckan.org", key=getOption("ckan_demo_key"))
+#'
+#' # create an organization
+#' (res <- organization_create("foobar", title = "Foo bars",
+#'   description = "love foo bars"))
+#'
+#' # delete the organization just created
+#' res$id
+#' organization_delete(id = res$id)
+#' }
```
Comment (sckott) [inline R/organization_purge.R:19]: i believe this brace is not matched. did you make sure to run check on this?

**Review by sckott — CHANGES_REQUESTED** [review]
Thanks! 

Can you add some brief to the docs for this fxn describing what purge means?

File: R/organization_purge.R (hunk 2 above)
Comment (nicholsn) [reply R/organization_purge.R]: thanks, ran check this time

**Comment by nicholsn** [comment]
@sckott thanks for the quick review! I added a blurb to the docs, let me know if you need more than that.

**Review by sckott — APPROVED** [review]

**Comment by sckott** [comment]
Thanks, looks good
